src/ResourceManager/Automation/Commands.Automation/ChangeLog.md

```diff
@@ -19,5 +19,8 @@
 -->
 ## Upcoming Release
 
+## Version 1.0.1
```

Comment:
Please remove the extra version header, this should just go under 'Upcoming Release'

Reply:
Done, just fixed it